nssp/README.md

```diff
@@ -0,0 +1,75 @@
+# NWSS wastewater data
+
+We import the wastewater data, currently only the smoothed concentration, from the CDC website, aggregate to the state and national level from the wastewater sample site level, and export the aggregated data.
```

Comment:
issue: Should reference NSSP, not WW data. Details of where data is coming from should also be checked.

Reply:
https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/39e2cbdec58d38be3d6b0321e40fb8a4bca58932